frontend/src/components/PlanHome/PlanHome.tsx

```diff
@@ -0,0 +1,130 @@
+import styles from './PlanHome.module.css'
+import editButton from '../../assets/editButton.png'
+import addMemberButton from '../../assets/addMemberButton.png'
+import { useParams } from 'react-router-dom'
+import { fetchPlanData } from '../Plan/hook/fetchPlanData'
+import { loadingStatus } from '../../utils/loadingStatus'
+import { fakeUserData } from '../PlanForm'
+import { MemberList } from './MemberList'
+
+import { buildStyles, CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+
+export const PlanHome = (): JSX.Element => {
+  const progressPercentage = 20;
+  const { id } = useParams();
+
+  const [, status] = fetchPlanData(id as string)
```

Comment:
Will come back to this later to work on the dynamic load

Reply:
let's add a todo then